### ISSUE TITLE
Changing specific modules' levels at runtime [JIRA: RIAK-1894]

### ISSUE THREAD

**Issue by dcheckoway** [erlang-lager/lager#231]
I'm trying to change the loglevel of specific modules at runtime -- really just once at app startup.  Here is what I've tried:

```
[lager:trace(lager_console_backend, [{module, Module}], notice) ||
     Module <- [shift,
                shift_connection,
                shift_store,
                shift_cache,
                shift_queue,
                shift_thrift_api_handler]]
```

...but that doesn't seem to be working.  It does, however, look like the filters are getting set up, as you can see here in these before/after lager.status() outputs:

https://gist.github.com/dcheckoway/ce2bbaec16838dd7deaf

But at runtime, the filtered levels are ignored.  Those modules for which I've set `notice` as the level are still logging `info` and `debug` messages, for example.  What am I doing wrong?

**Comment by mrallen1** [comment]
Should look into this for a future release.

**Comment by gnozell** [comment]
Is negative tracing still being looked into?

**Comment by mrallen1** [comment]
No active work is planned for it AFAIK.

**Comment by mrallen1** [comment]
We recently merged #389 which adds a bunch of new logical operators to filter capabilities exposing the functionality within the underlying goldrush library, which includes the notion of negative tracing as well as combining groups of filters with `AND` and `OR` and `NOT` operators. So I'm going to close this issue.

Please open a new issue if the new operators don't address your use case. Thanks.
